src/main/java/com/moneylendingapp/entities/User.java

```diff
@@ -36,7 +37,8 @@ public class User extends BaseEntity {
     @Enumerated(EnumType.STRING)
     private EmploymentStatus employmentStatus;
 
-
+    @Enumerated(EnumType.STRING)
+    private Role roles;
```

Comment:
It looks like it is just one role a user can have but the field is named "roles"